code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -408,7 +454,7 @@
 
 //////////////////////////////////////////////
 //                                          //
-//               REVS                 //
+//               REVS                       //
```

Comment:
i couldn't help myself

Reply:
Don't I hate all of these comments and want them dead